core/src/main/java/com/radixdlt/sync/LocalSyncService.java

```diff
@@ -430,10 +430,20 @@ private SyncState processSyncResponse(
     log.trace("LocalSync: Received sync response from {}", sender);
 
     if (!currentState.waitingForResponseFrom(sender)) {
-      log.warn("LocalSync: Received unexpected sync response from {}", sender);
+      log.trace("LocalSync: Received unexpected sync response from {}", sender);
+      metrics.sync().unexpectedSyncResponses().inc();
```

Comment:
_(if you have time for such nitpicking)_ If the metric had a `sender` label, then the log would not be needed at all.

Reply:
hmh, I'd prefer not to have it as a label, theoretically there isn't any upper limit of peers the node can talk to during its run lifetime